tests/em/fdem/forward/test_FDEM_analytics.py

```diff
@@ -61,8 +61,8 @@ def setUp(self):
         prb.pair(survey)
 
         try:
-            from pymatsolver import PardisoSolver
-            prb.Solver = PardisoSolver
+            from pymatsolver import Pardiso
```

Comment:
Actually, no, hmm. We could go back on this one? I also think we need some work on taking the `pymatsolver` stuff out.